internal/ciTools/ci.go

```diff
@@ -60,3 +67,24 @@ func SedBucketName(old, new string) error {
 
 	return nil
 }
+
+func DetokenizeCI(old, new, ciLocation string) {
```

Comment:
feel free to follow if it makes sense for you.

returning error in functions is nice since we can easily create unit tests for them. test will be like:
```
call DetokenizeCI(...)
if return error, fail, or you validate the expected error and so on.

logging is great, but I would log them outside in the function caller, and let the function only return the error. *ps: in some cases, is nice to log inside the function.

example:
err := DetokenizeCI(...)
if err != nil { log.Println(err) }